osu.Game.Tournament/Screens/Editors/TournamentEditorScreen.cs

```diff
@@ -70,6 +98,9 @@ private void load()
                 }
             });
 
+            if (IsSubScreen)
```

Comment:
rather than doing this, only construct and add the back button in this conditional.